extension/package.json

```diff
@@ -5,14 +5,16 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "check-coverage": "nyc check-coverage --lines 89",
+    "test-coverage-run": "c8 npm run test-run && npm run check-coverage",
+    "test-run": "npm run unit && npm run integration && npm run e2e",
+    "check-coverage": "c8 check-coverage --lines 89",
     "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
-    "cover:unit": "nyc --silent npm run unit",
-    "cover:integration": "nyc --silent --no-clean npm run integration",
-    "cover:e2e": "nyc --silent --no-clean npm run e2e",
-    "unit": "nyc mocha --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.js\"",
-    "integration": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/integration/**/*.spec.js\"",
-    "e2e": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/e2e/**/*.spec.js\"",
+    "cover:unit": "npm run unit",
```

Comment:
This command is not coverage anymore, right? Seems like redundant now. If so, remove it.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/976/commits/60686d66e0d61607425d090147573b5566060bc7